### DISCUSSION TITLE
How do i set a default branch in the releases screen?

### DISCUSSION THREAD

**Discussion by tc** [community/community#108331] [New to GitHub]
Our workflow is:
default develop branch where developers work off of.
Merge develop into `master` deploys to staging.
Git tag master to trigger prod deploy.

We want to do trigger and creation of the git tag from the `/releases/new` screen.

Can we set the default branch ONLY on the release screen to be master vs the "default branch"?

![Screenshot 2024-02-14 at 12 02 44 PM](https://github.com/community/community/assets/7976/5ceb6d33-7cda-4419-bca2-95cf3cdb7bc9)

We still want develop as the default branch since developers use that as the base

**Comment by NipulM** [comment]
Hello 👋🏼

Yes, you can achieve this workflow by configuring the default branch specifically for the release screen while keeping `develop` as the default branch. 

1. Go to your repository on GitHub.
Click on the "Settings" tab near the top-right corner of the repository page.

2. Navigate to Branches Settings:
In the Settings sidebar, click on "Branches".

3.Set Default Branch for Releases:
Scroll down to the "Default branch" section.
You should see an option to set the default branch for releases.
Select master as the default branch for releases.

By setting master as the default branch for releases in GitHub, whenever you create a new release via the /releases/new screen, GitHub will default to master as the branch for creating the release. Developers will still see develop as the default branch for regular development work.

Hope this answers your question! :)

**Reply by tc** [reply]
3.Set Default Branch for Releases:
Scroll down to the "Default branch" section.
You should see an option to set the default branch for releases.
Select master as the default branch for releases.

<img width="981" alt="Screenshot 2024-02-15 at 2 44 58 PM" src="https://github.com/community/community/assets/7976/e1786e98-3f73-40bb-a191-838e73207e25">

Where is the "option to set the default branch for releases"?

I only see Default Branch with no option?
